platform_channels/lib/src/accelerometer_event_channel.dart

```diff
@@ -15,23 +15,23 @@ class Accelerometer {
   static Stream<AccelerometerReadings> get readings {
     return _eventChannel.receiveBroadcastStream().map(
           (dynamic event) => AccelerometerReadings(
-            event[0] as double,
-            event[1] as double,
-            event[2] as double,
+            event[0] as double?,
+            event[1] as double?,
+            event[2] as double?,
           ),
         );
   }
 }
 
 class AccelerometerReadings {
   /// Acceleration force along the x-axis.
-  final double x;
+  final double? x;
```

Comment:
Are these ever null coming from the native side?

Reply:
I don't believe it can be null from native side. We do have checks at the native side, if ever it's the respective data object which provides accelerometer values is null, we throw error.

Reply:
Also @adarsh-technocrat can you please update the sample to throw an Exception if [sensorEvent is null](https://github.com/flutter/samples/blob/090e95b3a1f062eb5689dd20478f1fa0741b272e/platform_channels/android/app/src/main/kotlin/dev/flutter/platform_channels/AccelerometerStreamHandler.kt#L32)? We don't throw an exception at the iOS side. It would be good to align them.

Reply:
> Also @adarsh-technocrat can you please update the sample to throw an Exception if [sensorEvent is null](https://github.com/flutter/samples/blob/090e95b3a1f062eb5689dd20478f1fa0741b272e/platform_channels/android/app/src/main/kotlin/dev/flutter/platform_channels/AccelerometerStreamHandler.kt#L32)? We do throw an exception at the iOS side. It will be good to align them.

Sure @AyushBherwani1998  I'll update PR accordingly :)